wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+- **You cheated.** Using a third-party program, macros, or any form of unfair advantage will result in an immediate restriction.
+- **Other cases.** The accounts team reserves the right to restrict an account for cases that are not covered by the three above. This generally includes investigating allegations of cheating or punishing excessive bad behaviour on the forums or chats.
+
+## What do I do while on cooldown before appealing?
+- **Keep enjoying the game!** While your account is restricted and you are on cooldown, you are more than welcome to continue playing the game (as long as you play on your original account!). The scores you achieve while restricted will be submitted, and unless your account is rolled back, they will remain after you become unrestricted.
+- **Maintain good behaviour.** We understand that being restricted can be frustrating, but think of it as a chance to take some time off, away from the game and community. Try to avoid interacting with the community at least for a few days (whether it be via discord or social media) to cool down and reflect on the actions that may have contributed to your account being restricted. Raging at others, spamming the support team, or insulting the staff is not going to help in making you unrestricted.
+- **Make sure you do respect the rules.** Restrictions are punishments for breaking the rules. If you continue doing so while restricted, the cooldown before you can appeal will be reset. This includes continuing to cheat, or creating or accessing new accounts to evade the restriction.
+- **Follow the support team's instructions.** The three-month cooldown imposed on accounts is a chance to take time off and reflect on your actions. If you attempt appealing before that time, or continue trying to contact the support team to shorten the cooldown, you will be ignored.  If you are unsure when your cooldown expires, when you will be allowed to appeal, or which of your accounts you should be playing on and appealing for, you are welcome to send us an email to ask. Just make sure not to send any appeals before the date given to you!
+- **Do not hesitate to contact us if you have any questions.** As long as you are not trying to circumvent punishment or shorten the cooldown, we will happily answer any questions you have about your account's restriction!
+
+## Three months have passed. How do I appeal?
+
+You can appeal by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh). Again, we will need your username, and make sure that you are sending it from the email address attached to your account! After you send your appeal, expect a reply within one week. If you have not received one then, please contact us on twitter or by email - there is a chance we have not received it or closed it by accident!
```

Comment:
`You can appeal by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh). Again, we will need your username, and make sure to appeal from the email address attached to your account! After you send your appeal, expect a reply within one week. If you have not received one by then, please contact us on twitter or by email - there is a chance we have not received it or closed it by accident!`